crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -201,6 +201,22 @@ def compute_resources_available?
         true
       end
 
+      # Check for presence of HA setup, which is a requirement for non-disruptive upgrade
+      def ha_presence_check
```

Comment:
@rsalevsky if you want to discuss how the methods should look like and what kind of return values should they return, please comment rather https://github.com/crowbar/crowbar-core/pull/672 . I'll adapt this one according to changes in 672 once it is merged